docs/dev/user_management.rst

```diff
@@ -108,11 +108,32 @@ Space is reserved next to the roster for an information panel to display elabora
 selection. Right now it includes only the total # of students which match the criteria determined by the selectors
 and filters.
 
-Role Management
----------------
+Facility Management
+-------------------
 
-Role Management will provide an interface for assigning Coach and Admin roles to users.
+Facility Management (previously referred to as Role Management) will provide an interface for managing user Roles in
+a Facility. Users may multiply possess Coach and Admin roles for a Facility or Classrooms within a Facility.
```

Comment:
"Users may multiply possess Coach and Admin roles for a Facility or Classrooms within a Facility" - phrasing